src/settings-compat-v7/_font.scss

```diff
@@ -14,7 +14,7 @@
 $fw-semi-bold: 600 !default;
 
 // Font sizes
-$fz-base: 14 !default;
+$fz-base: 14px !default;
```

Comment:
This change looks good to me, but then, maybe we don't need the function `strip-unit` at https://github.com/SUI-Components/sui-theme/blob/86f2c7f06bba934061c9986ef89814b6ff7419f6/src/settings-compat-v7/_font.scss#L3

Reply:
what does the function strip-unit?